### ISSUE TITLE
TorchInductor OpInfo accuracy failure Tracker

### ISSUE THREAD

**Issue by SherlockNoMad** [pytorch/pytorch#93691]
Following OpInfo tests are failing due to accuracy issue:

To reproduce the error, run 
```
pytest -sv  test/test_torchinductor_opinfo.py -k test_comprehensive_<opname>_<device_type>_<dtype>
```

cc @ezyang @soumith @msaroufim @wconstab @ngimel @bdhirsh @anijain2305 

- [ ] _masked.argmax	cuda	torch.float16	
- [ ] _masked.argmax	cuda	torch.float32	
- [ ] _masked.argmax	cuda	torch.float64	
- [ ] _masked.argmax	cuda	torch.int32	
- [ ] _masked.argmax	cuda	torch.int64	
- [ ] _masked.argmin	cuda	torch.float16	
- [ ] _masked.argmin	cuda	torch.float32	
- [ ] _masked.argmin	cuda	torch.float64	
- [ ] _masked.argmin	cuda	torch.int32	
- [ ] _masked.argmin	cuda	torch.int64	
- [ ] _masked.cumprod	cuda	torch.float16	
- [ ] _masked.cumsum	cuda	torch.float16	
- [x] add	cuda	torch.bool	
- [ ] addbmm	cuda	torch.float16	
- [ ] addmv	cuda	torch.float16	
- [ ] addr	cuda	torch.float16	
- [x] argmax	cuda	torch.float16	(#859)
- [x] argmax	cuda	torch.float32	
- [x] argmax	cuda	torch.float64	
- [x] argmax	cuda	torch.int32	
- [x] argmax	cuda	torch.int64	
- [x] argmin	cuda	torch.float16	
- [x] argmin	cuda	torch.float32	
- [x] argmin	cuda	torch.float64	
- [x] argmin	cuda	torch.int32	
- [x] argmin	cuda	torch.int64	
- [ ] baddbmm	cuda	torch.float16	 
- [ ] cross	cuda	torch.float16	
- [ ] cumsum	cuda	torch.float16	
- [ ] diff	cuda	torch.float16	
- [ ] dist	cuda	torch.float16	
- [ ] linalg.cholesky	cuda	torch.float32	
- [ ] linalg.cholesky	cuda	torch.float64	
- [ ] linalg.cholesky_ex	cuda	torch.float32	
- [ ] linalg.cholesky_ex	cuda	torch.float64	
- [ ] linalg.cross	cuda	torch.float16	
- [ ] linalg.lu_factor	cuda	torch.float32	
- [ ] linalg.lu_factor	cuda	torch.float64	
- [ ] linalg.lu_factor_ex	cuda	torch.float32	
- [ ] linalg.lu_factor_ex	cuda	torch.float64	
- [ ] logcumsumexp	cuda	torch.float16	
- [x] logical_and	cuda	torch.int32	
- [x] logical_and	cuda	torch.int64	
- [x] logical_or	cuda	torch.int32	
- [x] logical_or	cuda	torch.int64	
- [ ] matrix_exp	cuda	torch.float16	
- [ ] max.reduction_with_dim	cuda	torch.int32	
- [ ] max.reduction_with_dim	cuda	torch.int64	
- [ ] min.reduction_with_dim	cuda	torch.int32	
- [ ] min.reduction_with_dim	cuda	torch.int64	
- [ ] nn.functional.binary_cross_entropy	cuda	torch.float16	
- [ ] nn.functional.multi_margin_loss	cuda	torch.float16	
- [ ] nn.functional.multilabel_margin_loss	cuda	torch.float16	
- [ ] nn.functional.soft_margin_loss	cuda	torch.float16	
- [ ] renorm	cuda	torch.float16	
- [ ] select_scatter	cuda	torch.int32	
- [ ] select_scatter	cuda	torch.int64	
- [x] sign	cuda	torch.bool	
- [ ] sign	cuda	torch.int32	
- [ ] sign	cuda	torch.int64

**Comment by desertfire** [comment]
Taking another look in https://github.com/pytorch/pytorch/issues/105534
